geomstats/geometry/special_linear.py

```diff
@@ -0,0 +1,241 @@
+"""Class for the group of special linear matrices."""
+
+import geomstats.algebra_utils as utils
+import geomstats.backend as gs
+from geomstats.geometry.invariant_metric import InvariantMetric
+from geomstats.geometry.lie_algebra import MatrixLieAlgebra
+from geomstats.geometry.lie_group import MatrixLieGroup
+from geomstats.geometry.matrices import Matrices
+
+
+class SpecialLinear(MatrixLieGroup):
+    """Class for the Special Linear group SL(n).
+
+    This is the space of invertible matrices of size n and unit determinant.
+    Parameters
+    ----------
+    n : int
+        Integer representing the shape of the matrices: n x n.
+    """
+
+    def __init__(self, n):
+        """Construct an instance of SpecialLinear()."""
+        super(SpecialLinear, self).__init__(
+            dim=int((n * (n - 1)) / 2),
+            representation_dim=n,
+            lie_algebra=TracelessMatrices(n=n),
+        )
+        self.metric = InvariantMetric(self)
+
+    def belongs(self, point, atol=gs.atol):
+        """Evaluate if a point belongs to the group.
+
+        Check the size and the value of the determinant.
+        Parameters
```

Comment:
Clean the docstring style.

See errors in GitHub action's linting: they ask you to add blank lines between the sections of a docstring.
Also see the ECE594n HW guidelines which show an example of how docstring should be formatted.